utils/node_local_test.py

```diff
@@ -3392,7 +3370,7 @@ def make_dirs(parent, count):
                 pass
             start = time.time()
             for i in range(count):
-                f = open(os.path.join(file_dir, str(i)), 'w')
+                f = open(join(file_dir, str(i)), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations